### PR TITLE
fix: use labelRange only line scatter chart

### PR DESCRIPTION
**Pull request by jung-han**
### Please check if the PR fulfills these requirements
- [ ] It's submitted to right branch according to our branching model
- [ ] It's right issue type on title
- [ ] When resolving a specific issue, it's referenced in the PR's title (e.g. `fix #xxx[,#xxx]`, where "xxx" is the issue number)
- [ ] The commit message follows our guidelines
- [ ] Tests for the changes have been added (for bug fixes/features)
- [ ] Docs have been added/updated (for bug fixes/features)
- [ ] It does not introduce a breaking change or has description for the breaking change

### Description

asis
image

tobe
image

use `labelRange` only line scatter chart

---
Thank you for your contribution to TOAST UI product. 🎉 😘 ✨

### CODE REVIEW

**Review by js87zz — APPROVED** [review]
리뷰 완료입니다!

File: apps/chart/src/store/axes.ts [hunk 1]
```diff
@@ -131,15 +131,15 @@ export function getLabelAxisData(stateProp: ValueStateProp): LabelAxisState {
...
   const pointOnColumn = isPointOnColumn(series, options);
   const dateType = isDateType(options, axisName);
   const labelsBeforeFormatting = isCoordinateTypeChart
     ? makeLabelsFromLimit(scale.limit, scale.stepSize, dateType)
     : categories;
   const labels = getLabelsAppliedFormatter(labelsBeforeFormatting, options, dateType, axisName);
   let labelRange;
-
-  if (scale) {
+  if (scale && hasLineSeries) {
```
Comment (jungeun-cho) [inline apps/chart/src/store/axes.ts:142]: labelRange는 라인시리즈에서만 사용하는 것이죠?~

**Review by lja1018 — APPROVED** [review]
리뷰완료합니다.

**Review by jungeun-cho — COMMENTED** [review]
스크린샷 2021-04-14 15 17 00
x축 tick과 plot line 확인부탁합니다.

**Review by shiren — APPROVED** [review]
수고하셨습니다.

**Review by jungeun-cho — APPROVED** [review]
수고하셨습니다.

**Pull request merged**